mitiq/cdr/cdr.py

```diff
@@ -166,3 +166,167 @@ def execute_with_cdr(
         p0=np.zeros(num_fit_parameters),
     )
     return fit_function(noisy_results[0, :], fitted_params)
+
+
+def mitigate_executor(
+    executor: Callable[[QPROGRAM], QuantumResult],
+    observable: Optional[Observable] = None,
```

Comment:
The `observable` argument is not documented in the docstring. Also in other functions.